index.html

```diff
@@ -42,7 +42,7 @@ <h1> <span>I’m Vishal</span><span class="text-span">Glad to see</span> <span
                 <h2>LET’S CONNECT</h2>
```

Comment:
- [ ] On line 8, add a more meaningful page title. You may choose to use Hector's portfolio instead of my portfolio